src/argilla/labeling/text_classification/rule.py

```diff
@@ -74,6 +78,34 @@ def author(self):
         """Who authored the rule."""
         return self._author
 
+    def convert_to_labeling_rule(self):
```

Comment:
this method must be private 

rename to `__convert_to_labeling_rule` and its references

Reply:
done